examples/images_contours_and_fields/multi_image.py

```diff
@@ -8,7 +8,7 @@
 It also illustrates colorbar tick labelling with a multiplier.
 """
 
-from matplotlib.pyplot import figure, show, axes, sci
+from matplotlib.pyplot import figure, show, sca, sci
```

Comment:
My only compalint is that I don't think we should be encouraging pyplot in the examples.

Reply:
Mildly agreed. But that's for another discussion. This one is about the `pyplot.axes()` API, so I just adapt the existing pyplot examples.